proto/Cargo.toml

```diff
@@ -19,7 +19,7 @@ prost-codec = ["prost-derive", "bytes", "lazy_static", "grpcio/prost-codec", "pr
 
 [dependencies]
 futures = "0.3"
-grpcio = { path = "..", features = ["secure"], version = "0.7.0", default-features = false }
+grpcio = { path = "..", features = ["secure"], version = "0.7.1", default-features = false }
```

Comment:
Can you also change the version of grpcio-proto to 0.7? I forgot to do so last time.

Reply:
Done